StrataVerify.lean

```diff
@@ -64,30 +66,47 @@ def main (args : List String) : IO UInt32 := do
       println! s!"Successfully parsed."
       if opts.parseOnly then
           return 0
-      let vcResults ← if file.endsWith ".csimp.st" then
-        C_Simp.verify "z3" pgm opts
-      else
-        verify "z3" pgm opts
-      for vcResult in vcResults do
-        println! f!"{vcResult.obligation.label}: {vcResult.result}"
-      let success := vcResults.all isSuccessVCResult
-      if success && !opts.checkOnly then
-        println! f!"Proved all {vcResults.size} goals."
-        return 0
-      else if success && opts.checkOnly then
-        println! f!"Skipping verification."
-        return 0
-      else
-        let provedGoalCount := (vcResults.filter isSuccessVCResult).size
-        let failedGoalCount := (vcResults.filter isFailureVCResult).size
-        println! f!"Finished with {provedGoalCount} goals proved, {failedGoalCount} failed."
-        return 1
+      else if opts.typeCheckOnly then
+        let ans := if file.endsWith ".csimp.st" then
+                    C_Simp.typeCheck pgm opts
```

Comment:
This is a super tiny thing, but the branches of this `if` are indented by only 1 space, rather than the 2 used elsewhere.